src/components/visualizations/common/utils/drilldownEventing.ts

```diff
@@ -0,0 +1,205 @@
+// (C) 2007-2018 GoodData Corporation
+import { get, debounce } from 'lodash';
+import * as invariant from 'invariant';
+import * as CustomEvent from 'custom-event';
+import { IDrillableItem } from '../../../../';
+import { AFM, Execution } from '@gooddata/typings';
+import { IDrillEventIntersectionElement } from '../../../../interfaces/DrillEvents';
+import { VisElementType, VisType, VisualizationTypes } from '../../../../constants/visualizationTypes';
+
+export interface IDrillableItemLocalId extends IDrillableItem {
+    localIdentifier: AFM.Identifier;
+}
+
+export function isDrillableItemLocalId(item: IDrillableItem | IDrillableItemLocalId): item is IDrillableItemLocalId {
```

Comment:
as `IDrillableItemLocalId` is extending from `IDrillableItem` this definition is not necessary 

```
item: IDrillableItemLocalId
```
is enough

Reply:
Actually it is needed to use both types.

Reply:
yes you are right :)